README.md

```diff
@@ -1,2 +1,54 @@
-# get-datetime-details-while-tapping-the-.NET-MAUI-Calendar
-This repository contains a sample of get datetime details while tapping the .NET MAUI Calendar.
+# How to get Datetime details while tapping the .NET MAUI Calendar
+
+This repository contains a sample to get the datetime details while tapping using the [Syncfusion .NET MAUI Calendar](https://help.syncfusion.com/maui/calendar/getting-started) control.
+
+Please refer the KB through this link.
+
+## Syncfusion controls
+
+This project used the following Syncfusion control(s):
+* [SfCalendar](https://www.syncfusion.com/maui-controls/maui-calendar)
+
+## Supported platforms
+
+.NET Multi-platform App UI (.NET MAUI) apps can be written for the following platforms:
+
+* Android 5.0 (API 21) or higher.
+* iOS 10 or higher.
+* macOS 10.13 or higher, using Mac Catalyst.
+* Windows 11 and Windows 10 version 1809 or higher, using [Windows UI Library (WinUI) 3](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/).
+
+## Requirements to run the sample
+
+* [Visual Studio 2022 Preview](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview) version 17.3.4 or higher (.NET MAUI version 6.0.486) or [Visual Studio 2022 for Mac Preview](https://visualstudio.microsoft.com/vs/mac/preview/) version 17.4 .
+* .NET 6.0
+
+Refer to the following link for more details: [System Requirements](https://help.syncfusion.com/maui/system-requirements)
+
+## How to run the sample
+
+1. Clone the sample and open it in Visual Studio 2022 Preview.
```

Comment:
1. Clone the sample and open it in Visual Studio 2022 preview.

Reply:
Changed the casing.